lib/iris/tests/unit/time/test_PartialDateTime.py

```diff
@@ -224,12 +224,12 @@ def setUp(self):
         self.expected_value = EQ_EXPECTATIONS
 
     def test_cftime_equal(self):
-        pdt = PartialDateTime(month=3, microsecond=2)
+        pdt = PartialDateTime(month=3, second=2)
         other = cftime.datetime(year=2013, month=3, day=20, second=2)
         self.assertTrue(pdt == other)
```

Comment:
This was failing because the equality operator returned `False`.  Possibly `cftime.datetime`s (or their predecessors) did not used to have a microseconds, but they do now.